src/lib/config_v2/config.go

```diff
@@ -0,0 +1,67 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"sync"
+	"time"
+
+	"gopkg.in/yaml.v3"
+)
+
+//Config struct stores the APIs parsed from local-config.yml file.
```

Comment:
```suggestion
// Config contains configuration in runtime.
```
Please follow the suggestions I have in previous reviews. It saves a lot of redundant work in reviewing.